src/main/java/soot/SootMethod.java

```diff
@@ -586,7 +586,7 @@ public boolean isStaticInitializer() {
    * @return yes, if this is a class initializer or main function.
    */
   public boolean isEntryMethod() {
-    if (isStatic() && subsignature.equals(VirtualCalls.v().sigClinit)) {
```

Comment:
we should delete `sigClinit` field completely

Reply:
@pavanupb could you please do this?

Reply:
@mbenz89 The sigClinit field in SootMethod class has been removed completely in my feature/default-methods branch. While merging I took the changes from develop, so after merging my PR this should be reflected.